data/transition-sites/phe_hpa.yml

```diff
@@ -9,3 +9,4 @@ aliases:
 - hpa.org.uk
 - www.hpa.nhs.uk
 - hpa.nhs.uk
+- legacytools.hpa.org.uk
```

Comment:
Does this domain serve the original content at the same paths as it was on www.hpa.org.uk, or was the content moved to different paths for the legacytools domain? It only makes sense to add this domain as an alias if the same mappings can be used for both. /cc @marksheldonGDS